index.d.ts

```diff
@@ -607,7 +607,6 @@ declare module 'warframe-items' {
         'Melee' |
         'OperatorAmps' |
         'Pistols' |
-        'SentinelWeapons' |
```

Comment:
Should be a Category not a ProductCategory

Reply:
it is a ProductCategory and a Category.

Reply:
I will make changes to reflect that properly.